<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerrainInfo</class>
 <widget class="QWidget" name="TerrainInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Height Map:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Length:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="txtLength">
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>1024</number>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QSpinBox" name="txtWidth">
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>1024</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Tile Length:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Width:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Tile Width:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QDoubleSpinBox" name="txtTileWidth">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="txtTileLength">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Load:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="txtLoad">
            <property name="text">
             <string>terrain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnLoad">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="MagicGameEditor.qrc">
              <normaloff>:/MagicGameEditor/Images/folder.png</normaloff>:/MagicGameEditor/Images/folder.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Save:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="txtSave">
            <property name="text">
             <string>terrain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnSave">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="MagicGameEditor.qrc">
              <normaloff>:/MagicGameEditor/Images/save.png</normaloff>:/MagicGameEditor/Images/save.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnEdit">
        <property name="toolTip">
         <string>Modify the Terrain</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="MagicGameEditor.qrc">
          <normaloff>:/MagicGameEditor/Images/terrain.png</normaloff>:/MagicGameEditor/Images/terrain.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRaise">
        <property name="toolTip">
         <string>Raise</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="MagicGameEditor.qrc">
          <normaloff>:/MagicGameEditor/Images/raise.png</normaloff>:/MagicGameEditor/Images/raise.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnLower">
        <property name="toolTip">
         <string>Lower</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="MagicGameEditor.qrc">
          <normaloff>:/MagicGameEditor/Images/lower.png</normaloff>:/MagicGameEditor/Images/lower.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnFlatten">
        <property name="toolTip">
         <string>Flatten</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="MagicGameEditor.qrc">
          <normaloff>:/MagicGameEditor/Images/flatten.png</normaloff>:/MagicGameEditor/Images/flatten.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="btnCircle">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="MagicGameEditor.qrc">
             <normaloff>:/MagicGameEditor/Images/circle.png</normaloff>:/MagicGameEditor/Images/circle.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnSquare">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="MagicGameEditor.qrc">
             <normaloff>:/MagicGameEditor/Images/square.png</normaloff>:/MagicGameEditor/Images/square.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Brush:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Radius:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QDoubleSpinBox" name="txtHeight">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-256.000000000000000</double>
        </property>
        <property name="maximum">
         <double>768.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="txtRadius">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>1024.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnUpdate">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>Update Terrain</string>
     </property>
     <property name="icon">
      <iconset resource="MagicGameEditor.qrc">
       <normaloff>:/MagicGameEditor/Images/clock.png</normaloff>:/MagicGameEditor/Images/clock.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnRaise</tabstop>
  <tabstop>btnLower</tabstop>
  <tabstop>btnFlatten</tabstop>
  <tabstop>btnCircle</tabstop>
  <tabstop>btnSquare</tabstop>
  <tabstop>txtLength</tabstop>
  <tabstop>txtWidth</tabstop>
  <tabstop>txtTileLength</tabstop>
  <tabstop>txtTileWidth</tabstop>
  <tabstop>txtLoad</tabstop>
  <tabstop>btnLoad</tabstop>
  <tabstop>txtSave</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>txtRadius</tabstop>
  <tabstop>txtHeight</tabstop>
  <tabstop>btnUpdate</tabstop>
 </tabstops>
 <resources>
  <include location="MagicGameEditor.qrc"/>
 </resources>
 <connections/>
</ui>
